index.html

```diff
@@ -78,30 +79,30 @@ <h2>Why?</h2>
       </section>
 
 
-      <section>
+      <section id="meet_the_team">
         <h2>Meet the team</h2>
-        <ul>
-          <li>
+        <ul id="meet_the_team_items">
+          <li id="profile_one">
```

Comment:
would just suggest taking out the id = "profile_one" etc as they are not used in the css .